packages/react-playground/razzle.config.js

```diff
@@ -34,16 +34,19 @@ module.exports = {
   },
   modifyJestConfig: ({ jestConfig }) => {
     /** @type {import('@jest/types').Config.InitialOptions} **/
-    const config = { ...jestConfig };
+    const config = {
```

Comment:
is there any chance we can share `razzle.config.js` by requiring razzle config object from the workspace root?

Reply:
hmm, reasonable. Since we're gonna refactor some of the js SKD as well as playground's code. Let't me add a issue for this.